posts/2023/05-28-random-favourites.md

```diff
@@ -0,0 +1,65 @@
+==title==
+Random favourites
```

Comment:
4 things that make programming a joy

Reply:
Thanks for the suggestion @alanvardy I'm going to stick with mine in the title, because it sounds more like me. But I like the idea of adding the number 4 so I incorporated that later.